### PR TITLE
removed support for .ts files in mod

### PR DESCRIPTION
**Pull request by jmplexi**
###  Summary

Removed support for .ts files in the mod.ts file.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/{project_slug}/blob/main/.github/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: src/mod.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const run = async function (functionDir: string) {
   // For the hosted runtime, we only support js/ts files named w/ the callback_id
   // They should already be bundled into single files as part of the package uploaded
   const resp = await DispatchPayload(payload, (functionCallbackId) => {
-    const supportedExts = ["js", "ts"];
+    const supportedExts = ["js"];
```
Comment (filmaj) [inline src/mod.ts:17]: If we don't expect to support multiple different filetypes anymore, then let's remove most of the code here: no point having an array of extensions, or looping over them, to return an array of files.

**Review by filmaj — CHANGES_REQUESTED** [review]
Left a couple of comments. Let's please also make sure to write tests for the `dispatch-payload.ts` module while we're here, since those seem to be missing.

File: src/mod.ts [hunk 2]
```diff
@@ -11,15 +11,10 @@ export const run = async function (functionDir: string) {
...
+  // For the hosted runtime, we only support js files named w/ the callback_id
   // They should already be bundled into single files as part of the package uploaded
   const resp = await DispatchPayload(payload, (functionCallbackId) => {
-    const supportedExts = ["js", "ts"];
-    const potentialFunctionFiles = supportedExts.map((ext) =>
-      `${functionDir}/${functionCallbackId}.${ext}`
-    );
-
-    return potentialFunctionFiles;
+    return [`${functionDir}/${functionCallbackId}`];
```
Comment (shapirone) [inline src/mod.ts:17]: We need to keep the `js` extension here on the end

**Review by shapirone — CHANGES_REQUESTED** [review]
looks like we accidentally dropped the extension

**Review by selfcontained — APPROVED** [review]
Nice, thanks for the tests too!

**Pull request merged**